Formula/o/openjdk@8.rb

```diff
@@ -123,25 +141,37 @@ def install
         --with-toolchain-type=clang
         --with-zlib=system
       ]
+      # Work around Xcode 16 bug: https://bugs.openjdk.org/browse/JDK-8340341
```

Comment:
Refer to issue #195456 for the fix.

P.S. [JDK-8340341](https://bugs.openjdk.org/browse/JDK-8340341)